FindGTest.cmake

```diff
@@ -0,0 +1,84 @@
+#.rst:
+# FindGtest
+# --------
+# Finds the gtest library
+#
+# This will define the following variables::
+#
+# GTEST_FOUND - system has gtest
+# GTEST_INCLUDE_DIRS - the gtest include directories
+# GTEST_LIBRARIES - the gtest libraries
+#
+# and the following imported targets:
+#
+#   Gtest::Gtest   - The gtest library
+
+if(ENABLE_INTERNAL_GTEST)
```

Comment:
this whole block is not needed, ENABLE_INTERNAL_GTEST exists only in kodi, not for addons

Reply:
Sure thing, updated.